sites/fast-website/src/app/components/feature-card/feature-card.template.ts

```diff
@@ -0,0 +1,14 @@
+import { html } from "@microsoft/fast-element";
+import { FeatureCard } from "./feature-card";
+
+export const FeatureCardTemplate = html<FeatureCard>`
+    <header>
+        <slot name="header"></slot>
+    </header>
+    <main>
+        <slot></slot>
+        <div>
+            <slot name="footer"></slot>
+        </div>
+    </main>
```

Comment:
There should only be one main element per page and I'm not sure how this is going to present to assistive technologies. Similarly, I don't know that this is the appropriate usage of the header html element. I think the content here is going to dictate the markup - IE, passing a h2 or h3 as the header and passing other semantic markup as body text. Let's swap those elements to divs to be safe.

Reply:
I'll get it switched up!

Reply:
I changed the header element to a div with a card_heading class and the main to a div with card_body class.  Sound reasonable and sufficiently semantic?